frontend/src/components/_Example/Example.js

```diff
@@ -0,0 +1,50 @@
+// @flow
```

Comment:
All this stuff is not used and can be deleted, right?

Reply:
yes